### PR TITLE
Add firehose metrics + alarms

### PR DESCRIPTION
**Pull request by mark-ship-it**
Add firehose metrics + alarms (limited to redshift delivery for now)
add support for import lambdas (IFunction)

### CODE REVIEW

File: src/aspect.ts [hunk 1]
```diff
@@ -18,11 +19,17 @@ export interface WatchfulAspectProps {
...
+   * @default true
+   */
+  readonly firehose?: boolean;
+
   /**
    * Automatically watch AWS Lambda functions in the scope.
    * @default true
    */
-  readonly lambda?: boolean;
+  readonly lambdaFn?: boolean;
```
Comment (voho) [inline src/aspect.ts:32]: Why is this changing? Some naming conflict?

Comment (mark-ship-it) [reply src/aspect.ts:32]: I believe I ran into issues because the variable is the same name as the import for the lambda package

**Review by voho — APPROVED** [review]
LGTM

**Comment by BenChaimberg** [comment]
Please separate the changes to the Lambda API into a different PR. Support for Firehose L2 is coming soon so this will need to be refactored. Will review at that time

**Comment by BenChaimberg** [comment]
keep open

**Comment by BenChaimberg** [comment]
keep open

**Comment by BenChaimberg** [comment]
Keep open x3

**Pull request closed without merging**